src/components/ModalWindow/ModalWindow.tsx

```diff
@@ -33,6 +39,9 @@ const ModalWindow: FC<IChange> = ({ type, message, date, expDate, idd }) => {
     ) {
       if (!type) {
         dispatch(addModalTodo(input));
+        if (filtValue === Сondition[0] || filtValue === Сondition[1]) {
+          dispatch(checker());
+        }
```

Comment:
enum Condition {active: "active", completed: "completed"}
filtValue - Condition.active

If you use Condition [0] - you don't know what Condition[0] means

Reply:
I used reverse enam here, according to the standard, numbering started with index 0, 1... . Therefore, referring to Direction[0] I take the string "active". It's a bad practice as I did ?

Reply:
I don't like, because it isn't clear what Condition[0] means at first glance, you need go to that enum and check

Reply:
Understood, I removed the call through reverse enum